### ISSUE TITLE
Option to hide empty metas on video cards

### ISSUE THREAD

**Issue by mf35** [fupdec/mediaChips#112]
Pretty self explanatory title.
I'm using currently 8 complex metas to tag my videos. So i'm showing them all on my video cards.
But sometimes some of them are empty because not relevant for some videos.

Could it be possible to put an option somewhere (like on the Videos tab options menu) where we can tell the video cards to not show metas with no data ?

Thank you :)

**Comment by fupdec** [comment]
Yes, that's a great idea. I think it is possible to generalize this option to "hide empty meta on ALL cards".
